sdk/RELEASE.md

```diff
@@ -1,5 +1,21 @@
 # Current Version (Still in Development)
 
+## Major Features and Improvements
+
+## Breaking Changes
+
+### For Pipeline Authors
+
+### For Component Authors
+
+## Deprecations
+
+## Bug Fixes and Other Changes
+
+## Documentation Updates
+
+# Current Version (2.0.0-alpha.4)
```

Comment:
nit: Just keep `2.0.0-alpha.4` (remove `Current Version` and the parentheses)

Reply:
Can you help fix `Current Version (2.0.0-alpha.3)` as well? to be consistent with all previous versions.

Reply:
Ok sure. Thanks for pointing these out! Done.